scripts/yul_coverage.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+set -e
+
+ROOT_DIR="$(dirname "$0")"/..
+
+for arg in "$@"; do
+  if [ "$arg" == "--no-stats" ]; then
+    NO_STATS=1
+  elif [ "$arg" == "--no-errors" ]; then
+    SHOW_NO_ERRORS=1
+  elif [ "$arg" == "--internal-compiler-errors" ]; then
+    SHOW_UNIMPLEMENTED_FEATURE_ERRORS=1
+  elif [ "$arg" == "--unimplemented-feature-errors" ]; then
+    SHOW_INTERNAL_COMPILER_ERRORS=1
+  elif [ "$arg" == "--other-errors" ]; then
+    SHOW_OTHER_ERRORS=1
+  elif [ "$arg" == "--list-files" ]; then
+    ONLY_LIST_FILES=1
+  else
+    echo "$(basename "${0}") [--no-stats] [--no-errors] [--internal-compiler-errors] [--unimplemented-feature-errors] [--other-errors] [--list-files]"
+    exit 0
+  fi
```

Comment:
I think a case statement would be perfect for readability here:

```suggestion
  case "$arg" in
    --no-stats)                     NO_STATS=1 ;;
    --no-errors)                    SHOW_NO_ERRORS=1 ;;
    --internal-compiler-errors)     SHOW_UNIMPLEMENTED_FEATURE_ERRORS=1 ;;
    --unimplemented-feature-errors) SHOW_INTERNAL_COMPILER_ERRORS=1 ;;
    --other-errors)                 SHOW_OTHER_ERRORS=1 ;;
    --list-files)                   ONLY_LIST_FILES=1 ;;
    *)
      echo "$(basename "${0}") [--no-stats] [--no-errors] [--internal-compiler-errors] [--unimplemented-feature-errors] [--other-errors] [--list-files]"
      exit 0
  esac  
```